deploy/decco/templates/decco-deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       labels:
         k8s-app: decco-operator
     spec:
-      priorityClassName: high-priority
+      priorityClassName: system-cluster-critical
```

Comment:
The reason I introduced `high-priority` was I felt that system-cluster-critical should be reserved by things that are truly critical to to functioning of Kubernetes (e.g. DNS, CNI controllers). I'll still approve this, but reluctantly.

Reply:
Ah, I see. That's totally valid; my reason for removing it was based on how the resource is managed. Use of this class leaked in to other projects that couldn't guarantee the class would be there so that dependency was totally dangling.

I'll add this priority class back but name it something particular to decco, just to imply that it shouldn't be used by other resources. The same thing can be done for as many components that need a priority class like this; it doesn't hurt anything to have a few extras.